client/src/components/commons/RecipeItem.jsx

```diff
@@ -3,15 +3,26 @@ import PropTypes from 'prop-types';
 import { Card, Image, Icon } from 'semantic-ui-react';
 import toastr from 'toastr';
 
-import defaultImage from '../../../images/default_image.jpg';
```

Comment:
your default export should match the name of your base file so as import e.g. import `defaultImage` from '../../../images/`defaultImage`.jpg'

Reply:
Try reading up on Airbnb JS style guide on naming.